.github/workflows/ci.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --no-fail-fast
+          args: --workspace --no-fail-fast
```

Comment:
@luizirber , any good reason for not using `--workspace` here? I think it is important to check everything independently of where it is been modified. Do you agree?